Formula/mono.rb

```diff
@@ -127,15 +159,30 @@ def install
     resource("fsharp").stage do
       # Temporary fix for use propper .NET SDK remove in next release
       inreplace "./global.json", "3.1.302", "3.1.405"
+
+      # Help .NET SDK run by providing path to libraries or disabling features
+      if OS.linux?
+        ENV["DOTNET_SYSTEM_GLOBALIZATION_INVARIANT"] = "1"
+        ENV["LD_LIBRARY_PATH"] = "#{Formula["openssl@1.1"].opt_lib}:#{HOMEBREW_PREFIX}/lib"
+      end
+
       with_env(version: "") do
-        system "./build.sh", "-c", "Release"
+        system "./build.sh", "--configuration", "Release"
       end
       with_env(version: "") do
         system "./.dotnet/dotnet", "restore", "setup/Swix/Microsoft.FSharp.SDK/Microsoft.FSharp.SDK.csproj",
                                               "--packages", "fsharp-nugets"
       end
       system "bash", buildpath/"packaging/MacSDK/fsharp-layout.sh", ".", prefix
     end
+
+    # Try to work around `brew bottle` error from leftover processes
+    # Error: Text file busy @ rb_sysopen - /home/linuxbrew/.linuxbrew/Cellar/mono/6.12.0.182/bin/mono-sgen
+    sleep 30 if OS.linux?
```

Comment:
This build worked this time, and `nuget` also passed its test.  Should we merge as is or do you want to update this?